### DISCUSSION TITLE
gene_position_data_frame contains Ensembl id's, is it possible to add gene names

### DISCUSSION THREAD

**Discussion by Saranya-Balachandran** [Katsevich-Lab/sceptre#144] [Q&A] [answered]
Dear team, 
Thank you for the tool. I would like to do the analysis with gene names as row names in my response matrix. It fails in the construct_cis_pairs() as the gene_position_data_frame contains Ensembl id's. Is it possible to use gene names as well?

Also it does not through an error with the trans pair construction.

Regards,
Saranya

**Comment by ekatsevi** [comment, marked as answer]
You should be able to use gene names if you provide your own `response_position_data_frame` in `construct_cis_pairs()`; please see the [documentation](https://katsevich-lab.github.io/sceptre/reference/construct_cis_pairs.html) of the latter function.
